packages/core/eslint.config.js

```diff
@@ -1,3 +1,2 @@
-/* eslint-disable @typescript-eslint/no-require-imports */
```

Comment:
I am confused that removing this rule does not seem to create any issue. Did you put it somewhere else?